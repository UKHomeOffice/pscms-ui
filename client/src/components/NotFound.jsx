import React from 'react';
import { useTranslation } from 'react-i18next';
import { Link } from 'react-navi';

const NotFound = () => {
  const { t } = useTranslation();
  return (
    <div className="govuk-width-container">
      <main className="govuk-main-wrapper govuk-main-wrapper--l" id="main-content" role="main">
        <div className="govuk-grid-row">
          <div className="govuk-grid-column-two-thirds">
            <h1 className="govuk-heading-l">{t('error.page-not-found.title')}</h1>
            <p className="govuk-body">{t('error.page-not-found.check-address')}</p>
            <p className="govuk-body">
              {t('error.page-not-found.paste-address')}{' '}
              <Link href="/" className="govuk-link">
                {t('error.page-not-found.redirect-link')}
              </Link>
              .
            </p>
          </div>
        </div>
      </main>
    </div>
  );
};

export default NotFound;
