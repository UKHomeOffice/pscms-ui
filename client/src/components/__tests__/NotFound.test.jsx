import React from 'react';
import { shallow } from 'enzyme';
import NotFound from '../NotFound';

describe('Not Found page', () => {
  it('renders without crashing', () => {
    const wrapper = shallow(<NotFound />);
    expect(wrapper.exists()).toBe(true);
  });

  it('matches snapshot', () => {
    const wrapper = shallow(<NotFound />);
    expect(wrapper).toMatchSnapshot();
  });
});
