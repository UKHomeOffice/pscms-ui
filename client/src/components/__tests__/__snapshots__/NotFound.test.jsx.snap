// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not Found page matches snapshot 1`] = `
<div
  className="govuk-width-container"
>
  <main
    className="govuk-main-wrapper govuk-main-wrapper--l"
    id="main-content"
    role="main"
  >
    <div
      className="govuk-grid-row"
    >
      <div
        className="govuk-grid-column-two-thirds"
      >
        <h1
          className="govuk-heading-l"
        >
          error.page-not-found.title
        </h1>
        <p
          className="govuk-body"
        >
          error.page-not-found.check-address
        </p>
        <p
          className="govuk-body"
        >
          error.page-not-found.paste-address
           
          <Link
            className="govuk-link"
            href="/"
          >
            error.page-not-found.redirect-link
          </Link>
          .
        </p>
      </div>
    </div>
  </main>
</div>
`;
